spec.md

```diff
@@ -586,18 +589,66 @@ schema being used.
           "40 concurrent connections"
         ]
       }
+    }, {
+      "name": "fake-deprecated-plan",
+      "id": "009aba-XXXX-XXXX-XXXX-dace631cd648",
+      "description": "This is deprecated.",
+      "free": false,
+      "deprecation": {
+        "description": "Don't use this anymore.",
+        "since": "2018-04-20",
+        "eol": "2020-02-28",
+        "alternatives": [{
+            "description": "Use this plan instead.",
+            "plan_id": "0f4008b5-XXXX-XXXX-XXXX-dace631cd648"
+          }
+        ]
+      }
     }]
   }]
 }
 ```
 
+##### Deprecation Object 
+
+| Response Field | Type | Description |
+| --- | --- | --- |
+| description | string | A short description of the deprecation. |
+| since | date string | ISO 8601 formatted date when the Service Broker deprecated the service or plan. |
```

Comment:
Do we need to mandate it include the full date, meaning timezone? Otherwise we don't know the exact time .

Reply:
some of these are required, I assume, so we should add `*` and the legend at the end.

Reply:
is there a better way to say this? like ISO 8601 with time and timezone?

Reply:
Maybe: An ISO 8601 formatted date and time, with time zone, when the Service Broker...
?

Reply:
works for me, thanks!